dashboard/app/controllers/aichat_controller.rb

```diff
@@ -63,6 +63,32 @@ def log_chat_event
     render(status: :ok, json: response_body)
   end
 
+  # params are studentUserId: number, currentLevelId: number, scriptId: number
+  # POST /aichat/log_chat_event
```

Comment:
Should this be a GET request where the params are provided as query params?

Also nit: update path name?